orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/StartStageHandler.kt

```diff
@@ -76,6 +76,7 @@ open class StartStageHandler
             val exceptionDetails = exceptionHandlers.shouldRetry(e, stage.getName())
             if (exceptionDetails?.shouldRetry ?: false) {
               log.warn("Error planning ${stage.getType()} stage for ${message.executionType.simpleName}[${message.executionId}]")
+              message.setAttribute(MaxAttemptsAttribute(5))
```

Comment:
WRT comments about mutability elsewhere this is where I'd expect to see something more like

    queue.push(message.copy(maxAttempts = MaxAttempsAttribute(5)), retryDelay)